backend/app/services/group_service.py

```
@@ -1101,6 +1101,71 @@ def invite_all_users(
+        user_id: Current user ID for permission filtering
+        user_role: User's system role ('admin' or 'user')
+
+    Returns:
+        List of GroupResponse for child groups the user can access
+    """
+    # Query all active subgroups
+    subgroups = (
+        db.query(Namespace)
+        .filter(
+            Namespace.name.like(f"{parent_name}/%"),
+            Namespace.is_active == True,
```

Ruff (0.15.17)

[error] Avoid equality comparisons to `True`; use `Namespace.is_active:` for truth checks

Replace with `Namespace.is_active`

(E712)